common/djangoapps/student/models_api.py

```diff
@@ -46,3 +47,7 @@ def create_manual_enrollment_audit(
 
 def get_course_enrollment(user, course_run_key):
     return _CourseEnrollment.get_enrollment(user, course_run_key)
+
+def get_phone_number(user):
+    student = UserProfile.objects.get(user_id=3)
```

Comment:
Did you mean to hard code the user_id here?

Reply:
You should surround this with a Try catch.  If the user you are searching for does not exist a `DoesNotExist` exception will be thrown.